<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_llm_config</class>
 <widget class="QDialog" name="edit_llm_config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit LLM Config</string>
  </property>
  <widget class="QGroupBox" name="gb_add_llm_config">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>LLM Config</string>
   </property>
   <widget class="QLabel" name="label_model">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>57</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Model</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_model">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Model Name, ex) gpt-4o-mini, gemini-1.5-flash</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_api_key">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>57</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>API Key</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_api_key">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>API Key</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_api_type">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>57</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>API Type</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_api_type">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>API Type, ex) google, openai</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_price">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Price ($/M tokens, optional)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_price_input">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Input price</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_price_output">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>150</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Output price</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_base_url">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Base URL (for local LLM, optional)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_base_url">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>http://0.0.0.0:4000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tags">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Tags (seperated by ',',optional)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_tags">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>ex) local, llama3</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_edit_llm_config">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>230</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_llm_config">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>391</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
